app/code/Magento/Config/Model/Config/Source/Date/Short.php

```diff
@@ -5,23 +5,55 @@
  */
 namespace Magento\Config\Model\Config\Source\Date;
 
+use IntlDateFormatter;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Locale\ResolverInterface;
+
 /**
  * @api
  * @since 100.0.2
  */
 class Short implements \Magento\Framework\Option\ArrayInterface
 {
+    /**
+     * @var IntlDateFormatter
+     */
+    private $dateFormatter;
+
     /**
      * @return array
      */
     public function toOptionArray()
     {
         $arr = [];
         $arr[] = ['label' => '', 'value' => ''];
-        $arr[] = ['label' => strftime('MM/DD/YY (%m/%d/%y)'), 'value' => '%m/%d/%y'];
-        $arr[] = ['label' => strftime('MM/DD/YYYY (%m/%d/%Y)'), 'value' => '%m/%d/%Y'];
-        $arr[] = ['label' => strftime('DD/MM/YY (%d/%m/%y)'), 'value' => '%d/%m/%y'];
-        $arr[] = ['label' => strftime('DD/MM/YYYY (%d/%m/%Y)'), 'value' => '%d/%m/%Y'];
+        $arr[] = ['label' => 'MM/DD/YY ' . $this->getTimeFormat(time(), '(M/d/y)'), 'value' => 'M/d/y'];
```

Comment:
`value` could not be changed. This format seems to be specific to `strftime()` method and value expected to be saved for latter use(with this method ?). So existing code may produce unexpected result with newly saved values
I believe we may mark it as `deprecated` to avoid future use of format of deprecated function.
@fascinosum What do you think ? ^^

Something like `'MM/DD/YY (' . (new \DateTime())->format('m/d/y') . ')';` could be used to output label as it was but without use of deprecated method

Reply:
@xmav, code is fixed, please check if everything is well.

Thank you

Reply:
@xmav, code is fixed, please check if everything is well.

Thank you